templates/core/terraform/firewall.tf

```diff
@@ -1,6 +1,6 @@
 resource "azurerm_public_ip" "fwpip" {
   name                = "pip-fw-${var.tre_id}"
-  resource_group_name = var.resource_group_name
```

Comment:
I wouldn't touch firewall too much as it is extracted out in this PR: https://github.com/microsoft/AzureTRE/pull/1226

Reply:
Thanks, will remove once merged

Reply:
reverted